libs/core/src/components/pds-checkbox/pds-checkbox.scss

```diff
@@ -65,8 +65,10 @@ input {
   background-color: var(--pine-color-secondary);
   border: var(--pine-border);
   border-radius: var(--pine-dimension-2xs);
+  flex: none;
```

Comment:
Do we need to overwrite this or could it possibly be removed here and from the host and applied to the new container to handle it?

Reply:
My comment no longer applies after our convo. Marking as resolved.